test_runner/regress/test_unlogged.py

```diff
@@ -26,7 +28,12 @@ def test_unlogged(neon_simple_env: NeonEnv):
     conn2 = endpoint2.connect()
     cur2 = conn2.cursor()
     # after restart table should be empty but valid
-    cur2.execute("PREPARE iut_plan (int) AS INSERT INTO iut VALUES ($1)")
+    cur2.execute("PREPARE iut_plan (int) AS INSERT INTO iut (id) VALUES ($1)")
     cur2.execute("EXECUTE iut_plan (43);")
     cur2.execute("SELECT * FROM iut")
-    assert cur2.fetchall() == [(43,)]
+    results = cur2.fetchall()
+    # Unlogged sequences didn't exist in <= 14
+    if env.pg_version <= PgVersion.V14:
+        assert results == [(43, 2)]
+    else:
+        assert results == [(43, 1)]
```

Comment:
Hmm, so on v14, the sequence created for the GENERATED ALWAYS AS IDENTITY column is logged, and hence it keeps the old value, 2. While on v15 and above, it's unlogged so it resets to 1 on restart. Maybe explain that in a comment:

```suggestion
    # Unlogged sequences were introduced in v15. On <= v14, the sequence created for
    # the GENERATED ALWAYS AS IDENTITY column is logged, and hence it keeps the old
    # value (2) on restart. While on v15 and above, it's unlogged, so it gets reset to 1.
    if env.pg_version <= PgVersion.V14:
        assert results == [(43, 2)]
    else:
        assert results == [(43, 1)]
```